i18n/en.toml

```diff
@@ -29,7 +29,7 @@ other = "Access our team drive if you'd like to take a look at the Istio technic
 other = "If you'd like to contribute to the Istio project, consider participating in our working groups"
 
 [slack_desc]
-other = "Interactively discuss development issues with the Istio community on Slack (invitation-only)"
+other = "Interactively discuss development issues with the Istio community on Slack (Developers only, see FAQ)"
```

Comment:
could we link to the FAQ?

Reply:
Unfortunately not, as this is a HTML tooltip (that appears at footpage).

Reply:
got it, thanks!